index.js

```diff
@@ -0,0 +1,87 @@
+import Methods from './modules/methods.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
```

Comment:
- [ ] According to the requirements, The `luxon` Package should be added as a `.js` file to the project (in full or minified version), you should download the script and save it to your project. then Import it as a module. 👍
> Download it from [here](https://moment.github.io/luxon/#/install?id=es6) 
```js
import { DateTime } from "luxon";
```

![image](https://user-images.githubusercontent.com/45231276/199157118-0075cc68-3e92-419a-a862-84183f226135.png)


> To uninstall the `luxon` from `dependencies` 💡
```
 npm uninstall luxon 
```